main.tf

```diff
@@ -2653,6 +2653,80 @@ module "vpa" {
   tags = var.tags
 }
 
+
+################################################################################
+# Fargate Fluentbit
+################################################################################
+# Help on Fargate Logging with Fluentbit and CloudWatch
+# https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html
+locals {
+  fargate_fluentbit_cwlog_group         = "/${var.cluster_name}/fargate-fluentbit-logs"
+  fargate_fluentbit_cwlog_stream_prefix = "fargate-logs-"
+  fargate_fluentbit_default_config = {
+    output_conf  = <<-EOF
+    [OUTPUT]
+      Name cloudwatch_logs
+      Match *
+      region ${local.region}
+      log_group_name ${local.fargate_fluentbit_cwlog_group}
+      log_stream_prefix ${local.fargate_fluentbit_cwlog_stream_prefix}
+      auto_create_group true
+    EOF
+    filters_conf = <<-EOF
+    [FILTER]
+      Name parser
+      Match *
+      Key_Name log
+      Parser regex
+      Preserve_Key True
+      Reserve_Data True
+    EOF
+    parsers_conf = <<-EOF
+    [PARSER]
+      Name regex
+      Format regex
+      Regex ^(?<time>[^ ]+) (?<stream>[^ ]+) (?<logtag>[^ ]+) (?<message>.+)$
+      Time_Key time
+      Time_Format %Y-%m-%dT%H:%M:%S.%L%z
+      Time_Keep On
+      Decode_Field_As json message
```

Comment:
Separate these out to specific config keys in the `fargate_fluentbit` input map and use `try(...)` to provide default values instead of using `merge()`